sources/Interop/Windows/um/shellapi/SHELLEXECUTEINFOW.Manual.cs

```diff
@@ -0,0 +1,283 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/shellapi.h in the Windows SDK for Windows 10.0.20348.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System.Runtime.InteropServices;
+
+namespace TerraFX.Interop
+{
+    [StructLayout(LayoutKind.Explicit)]
+    public unsafe partial struct SHELLEXECUTEINFOW
+    {
+        public static uint Size
```

Comment:
maybe call this `SizeOfValue` ? or `ValueSize` ?

Reply:
What about just `SizeOf`?